### PR TITLE
Prepare v2alpha4

### PR DESCRIPTION
**Pull request by dgageot**
Signed-off-by: David Gageot <david@gageot.net>

### CODE REVIEW

**Comment by balopat** [comment]
Do you have a config change in the next two weeks that motives the bump?

**Comment by dgageot** [comment]
@balopat not that I know. I was trying to see if it worked for when we are going to need it. And it's not straightforward. See https://github.com/GoogleContainerTools/skaffold/issues/3633

**Comment by balopat** [comment]
Sounds good, yes we should fix the script. I just want to avoid merging this and causing unnecessary version bumps! Thank you for the exploration! Let's wait with merging this until we have a config change!

File: docs/content/en/schemas/v2alpha3.json [hunk 1]
```diff
@@ -1606,7 +1606,7 @@
           "type": "array",
           "description": "path to Kustomization files.",
           "x-intellij-html-description": "path to Kustomization files.",
-          "default": "[\".\"]"
```
Comment (tejal29) [inline docs/content/en/schemas/v2alpha3.json]: this change caused #3618
https://github.com/GoogleContainerTools/skaffold/pull/3620/files

**Comment by loosebazooka** [comment]
Meant to ping this last week. Can we merge this? There are some changes queued that require it now.

File: docs/content/en/schemas/v2alpha3.json (hunk 1 above)
Comment (nkubala) [reply docs/content/en/schemas/v2alpha3.json]: @dgageot should this be removed?

**Comment by dgageot** [comment]
@nkubala @loosebazooka This should be good now

**Pull request merged**